agent/engine/docker_image_manager.go

```diff
@@ -303,21 +309,145 @@ func (imageManager *dockerImageManager) removeUnusedImages(ctx context.Context)
 	imageManager.updateLock.Lock()
 	defer imageManager.updateLock.Unlock()
 
-	imageManager.imageStatesConsideredForDeletion = imageManager.imagesConsiderForDeletion(imageManager.getAllImageStates())
+	var ECSImagesDeletedTimes int
+	imageManager.imageStatesConsideredForDeletion = imageManager.imagesConsiderForDeletion(imageManager.getAllImageStates(), ctx)
 	for i := 0; i < imageManager.numImagesToDelete; i++ {
 		err := imageManager.removeLeastRecentlyUsedImage(ctx)
+		ECSImagesDeletedTimes = i + 1
```

Comment:
Is this for counting how many ecs managed images already been removed? Maybe change it to **ECSImagesDeletedNum**? "Times" is a little bit confusing.